pkgs/applications/video/kodi/addons/trakt-py/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildKodiAddon, fetchzip, addonUpdateScript, requests, six, arrow }:
+buildKodiAddon rec {
+  pname = "trakt-py";
```

Comment:
Since all the addons are `python` should we call this something like `trakt-module` or `trakt-lib` instead? It isn't really important... but I thought I would mention :man_shrugging: